src/components/notifications/BetaBanner.vue

```diff
@@ -56,7 +56,7 @@ export default {
       );
       if (!featureQ.exists()) {
         await context.root.$toasted.error(
-          context.root.$t('~~Failed to opt in!'),
+          context.root.$t('info.error_unable_switch_phone_system'),
```

Comment:
Kinda purposely kept this generic, as the "BetaBanner" component can be reused for any future beta features.

So I guess we should dynamically pass the error message. No change needed on your part, just giving heads up.